docs/guides/LiveCode Builder Language Reference.md

```diff
@@ -554,8 +554,11 @@ non-Windows platform then it is taken to be `default`.
 
 The Java binding string has the following form:
 
-    "java:[className>][functionType.]function[!calling]"
-    
+    "java[thread]:[className>][functionType.]function[!calling]"
```

Comment:
Is it worth considering putting the thread affinity somewhere else? Perhaps a final ?<thread-affinity> clause at the end.